src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+const Calculator = () => (
+  <div className="calculator">
+    <div className="display">0</div>
+    <div className="button-row">
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button">÷</button>
+    </div>
+    <div className="button-row">
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button">x</button>
+    </div>
+    <div className="button-row">
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button">-</button>
+    </div>
+    <div className="button-row">
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button">+</button>
+    </div>
+    <div className="button-row bottom-row">
+      <button type="button">0</button>
+      <button className="narrow" type="button">.</button>
+      <button className="narrow" type="button">=</button>
+    </div>
+  </div>
+);
```

Comment:
- [ ] Good job so far in your implementation, however, there should have at least one child component inside the Calculator.js component. You could either extract the calculator's result (line #3) into a separate piece and use it here or create a button component to be reused. The goal is to have at least a child component being used inside this component in which you'll pass props to it. Kindly work on that 👍🏽

<img width="971" alt="Screenshot 2023-08-18 at 11 13 36 AM" src="https://github.com/firepicaso/mm_calculator/assets/50941074/76bed497-c820-4be7-af30-decc235a7f08">